core/app/components/forms/types.ts

```diff
@@ -1,4 +1,7 @@
+import type { CoreSchemaType } from "db/public";
+
 export interface ElementProps {
-	label: string;
 	name: string;
+	config: any;
```

Comment:
is it possible to type this? since it seems like it gets used downstream quite a bit

Reply:
i could type it as unknown (probably?) but i'm not sure we can do better than that since it's a json object whose structure varies depending on the component. this is kind of a big drawback, so i am definitely not sure this is the right approach. the other big issue i foresee is that we'll need to be quite cautious about breaking changes to these schemas.